debian/changelog

```diff
@@ -1,3 +1,15 @@
+ubuntu-advantage-tools (27.0~21.10.1) impish; urgency=medium
```

Comment:
With the current devel release we don't ~XX.YY.1) so it can be just the following:
```suggestion
ubuntu-advantage-tools (27.0) impish; urgency=medium
```